### ISSUE TITLE
VEI meta/iast conversion

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/Cologne#211]
This issue for comments related to the meta-line/iast conversion of vei.txt, the Cologne digitization of the book `The Vedic Index of Names and Subjects`.

**Comment by funderburkjim** [comment]
### IAST issues

The text  uses Latin alphabet with diacritics for all Sanskrit words.  The conventions of the
text agree with modern IAST, with one minor extension:
* The l-dot character `ḷ`  is used for the Devanagari consonant ळ; it does not appear as the vowel ऌ

**Comment by funderburkjim** [comment]
### Footnote reorg
There are many footnotes in this text (~11,000). The original coding of these was like in KRM, and I changed
the coding in vei in a way similar to that which was done in KRM.  

Here is an example comparing the current display to the printed text:

Current display:
![image](https://user-images.githubusercontent.com/6393033/36358029-3f366dd0-14d5-11e8-8942-adc189b4fc0c.png)

Printed text:
![image](https://user-images.githubusercontent.com/6393033/36358036-57ea7dd0-14d5-11e8-9552-44c514be9a2b.png)

**Comment by funderburkjim** [comment]
### Sections of the text.

The digitization covers all parts of the two volumes of the text.  Here are the sections of vei.txt.
The numbers at the start of each line represent the line number within the digitization
```
      5:; ENTRIES VOLUME 1
  33222:; TITLE VOLUME 1
  33265:; FOREWORD
  33322:; PREFACE
  33726:; ENTRIES VOLUME 2
  64725:; ADDITIONS AND CORRECTIONS
  65053:; SANSKRIT INDEX
  70350:; ENGLISH INDEX
  72601:; MAP OF VEDIC INDIA
```

The Sanskrit Index and English Index Sections would be useful to add in some manner to the
Search capabilities.   Not only does the Sanskrit Index provide references to non-headwords, it also
provides very brief glosses for the words.  Here is a sample:
![image](https://user-images.githubusercontent.com/6393033/36358087-428989b2-14d6-11e8-813d-fc1d920e7d9c.png)
In other words, the authors have prepared a curated search facility in the two indexes, which it would
be nice to make use of in some way.

The [VEI Documentation](http://www.sanskrit-lexicon.uni-koeln.de/scans/csldoc/dictionaries/prefaces/veipref.html) provides a view of the foreword and preface.

**Comment by funderburkjim** [comment]
### Markup particulars
These comments apply to the entries only. The markup conventions of the other parts of the
text remain, for the most part, as in the original form of the digitization.

#### `<div n="X"/>`
This markup can have X as
* lb   typical line break.  The digitization followis the line breaks of the printed text
* P   a line break with indentation of the first line (585 instances)

Note that  this 'div' markup consists of empty xml tags.

#### `<sup>N</sup>`
Footnote references are annotated with a superscript number.

#### `<F>N) X</F>`
Footnote text X for footnote number N.

#### `<lang n="greek"></lang>`
About 100 instances.  The Greek text is not currently coded in vei.txt.

#### `<C n="N"/>`
Indicates a column in a tabular part of the text.  Used here:
![image](https://user-images.githubusercontent.com/6393033/36358156-5cded67c-14d7-11e8-9f3a-ce83090da555.png)

**Comment by funderburkjim** [comment]
The meta/iast conversion of vei.txt has now been installed at Cologne.

**Comment by gasyoun** [comment]
Give Greek to Greek! Nakshatra screenshot just mesmerising and welcomes me to explore the book deeper.
